gcc/rust/ast/rust-ast-full-test.cc

```diff
@@ -4093,8 +4101,7 @@ Module::process_file_path ()
   if (no_candidates_found || multiple_candidates_found)
     return;
 
-  module_file = file_mod_found ? expected_file_path
-			       : current_directory_name + expected_dir_path;
+  module_file = std::move (file_mod_found ? file_mod_path : dir_mod_path);
```

Comment:
Now this more accurately represents what was actually tested above with `file_exists` :) .